packages/gtk3.rb

```diff
@@ -62,13 +63,12 @@ class Gtk3 < Package
   depends_on 'libxrandr' # R
   depends_on 'mesa' => :build
   depends_on 'pango' # R
-  depends_on 'rest' # R
+  depends_on 'rest' => :build
   depends_on 'shared_mime_info' # L
   depends_on 'sommelier' # L
```

Comment:
Can we put sommelier last?

Reply:
Can we also add `depends_on 'xdg_base'` and replace the following:
```
    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/gtk-3.0"
    File.write("#{CREW_DEST_PREFIX}/etc/gtk-3.0/settings.ini", @gtk3settings)
```
with:
```
    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/.config/gtk-3.0"
    File.write("#{CREW_DEST_PREFIX}/.config/gtk-3.0/settings.ini", @gtk3settings)
```
?

Reply:
See https://developer-old.gnome.org/gtk3/stable/GtkSettings.html.

Reply:
I sort the list alphabetically with `sort -u` in my editor. Does it matter if sommelier is at the bottom? That's not going to affect the order of the installation of dependencies...

Reply:
I'm just thinking about the postinstall message that will scroll past the screen from newly installed gui apps.

Reply:
> I'm just thinking about the postinstall message that will scroll past the screen from newly installed gui apps.

Yes, but if we want sommelier last, I think we should address that by modifying crew to make the sommelier post-install show up last, since installing sommelier last definitely shouldn't break anything.

Reply:
> Can we also add `depends_on 'xdg_base'` and replace the following:
> 
> ```
>     FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/gtk-3.0"
>     File.write("#{CREW_DEST_PREFIX}/etc/gtk-3.0/settings.ini", @gtk3settings)
> ```
> 
> with:
> 
> ```
>     FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/.config/gtk-3.0"
>     File.write("#{CREW_DEST_PREFIX}/.config/gtk-3.0/settings.ini", @gtk3settings)
> ```
> 
> ?

Making that change... but xdg_base is already in the depends...

Reply:
Change made.